scss/_patterns_section-and-block.scss

```diff
@@ -0,0 +1,21 @@
+@mixin vf-container-spacing-vertical($spacing) {
+  margin-bottom: calc($spacing / 2) !important;
+
+  @media (min-width: $breakpoint-large) {
+    margin-bottom: calc($spacing) !important;
+  }
+}
+
+@mixin vf-p-section {
+  .p-block {
+    @include vf-container-spacing-vertical($spv--strip-shallow);
+  }
+
+  .p-section {
+    @include vf-container-spacing-vertical($spv--strip-regular);
+  }
+
+  .p-section--deep {
+    @include vf-container-spacing-vertical($spv--strip-deep);
+  }
+}
```

Comment:
We need to enumerate all elements that have a padding nudge from the top, and a margin-bottom compensation - that's `p, h6, h5, h4, h3, h2, h1, label, ...` etc - and adjust the values to be the current nudged values + the values above.

Reply:
@lyubomir-popov I'm not sure what do you mean, this is supposed to be a wrapper around the content, not a class name you put on heading or paragraph. 

So it's meant to be used as:

```
<section class="p-section">
   <div class="p-block">
      <h2>heading</h2>
      <p>some text…</p>
   </div>
   <div class="p-block">
       <p>another sibling block…</p>
    </div>
</section>
```

And NOT like:

```
<h2 class="p-block">Heading block…</h2>
```

Reply:
oh sorry I thought I saw u-sv--deep